bombastic/api/src/server.rs

```diff
@@ -218,17 +220,17 @@ impl From<&SearchParams> for SearchOptions {
 ///
 /// See the [documentation](https://docs.trustification.dev/trustification/user/retrieve.html) for a description of the query language.
 #[utoipa::path(
-    get,
-    tag = "bombastic",
-    path = "/api/v1/sbom/search",
-    responses(
-        (status = 200, description = "Search completed"),
-        (status = BAD_REQUEST, description = "Bad query"),
-        (status = 401, description = "Not authenticated"),
-    ),
-    params(
-        ("q" = String, Query, description = "Search query"),
-    )
+get,
+tag = "bombastic",
+path = "/api/v1/sbom/search",
+responses(
+(status = 200, description = "Search completed"),
+(status = BAD_REQUEST, description = "Bad query"),
+(status = 401, description = "Not authenticated"),
+),
+params(
+("q" = String, Query, description = "Search query"),
+)
```

Comment:
You don't like the indenting?

Reply:
Indeed, that's intellij messing things up sometimes. I will fix this.